api/envoy/extensions/filters/http/waf/v3/waf.proto

```diff
@@ -0,0 +1,223 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.waf.v3;
```

Comment:
I would suggest we do v3alpha here for now until we get a bit of experience with the filter. Then we can upgrade it to v3 later.